ui/packages/theme/src/reset.css

```diff
@@ -177,7 +177,7 @@ ul,
 menu {
 	margin: 0;
 	padding: 0;
-	list-style: none;
+	/* list-style: none; */
```

Comment:
Why comment it out, instead of just removing it altogether?

Reply:
forgot.